charters/PM/charter.md

```diff
@@ -0,0 +1,60 @@
+# PM Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the Data Coordination Platform (DCP) *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
```

Comment:
To Tactical responsibilities, add monitoring user metrics?

Reply:
WFM